chat-plugins/info.js

```diff
@@ -165,7 +165,8 @@ exports.commands = {
 	ipsearchall: 'ipsearch',
 	hostsearch: 'ipsearch',
 	ipsearch: function (target, room, user, connection, cmd) {
-		if (!target.trim()) return this.parse('/help ipsearch');
+		if (!target.trim() || !target.includes('.')) return this.parse('/help ipsearch');
```

Comment:
Should probably use net.isIP / net.isIPv4 / net.isIPv6. I don't recall if isIP exists as Node.js API